_posts/2015-08-24-Cocoa-Architecture:-Hybrid-Apps.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: 'Cocoa Architecture: Hybrid Apps'
+date: 2015-08-24T00:00:00.000Z
+comments: false
+categories:
+  - ios
+  - mobile
+  - architecture
+  - eigen
+  - hybrid
+author: Orta Therox
+github-url: 'https://www.github.com/orta'
+twitter-url: 'http://twitter.com/orta'
+blog-url: 'http://orta.io'
+---
+
+As a part of going through the design patterns we've found in the creation of the Artsy iOS apps, I'd like to talk a bit about Hybrid Applications. A hybrid application refers to an app that uses native code and web content intertwined. Our flagship iOS app, [eigen](https://github.com/artsy/eigen) is a hybrid app, and it seems to get more and more hybrid-y each release. Let's talk a little bit about the pros and cons of this. <!-- more -->
```

Comment:
"Hybrid Applications" is fine as a proper noun, but other uses of it should have the same capitalization. Maybe use `_Hybrid Applications_` instead? 

"hybrid-y" is _awesome_. 

"the pros and cons of this" -> "the pros and cons of this approach."